packages/framework/block-std/src/view/element/lit-host.ts

```diff
@@ -58,7 +58,10 @@ export class EditorHost extends SignalWatcher(
       return html`${nothing}`;
     }
 
-    const tag = view.component;
+    const tag =
+      typeof view.component === 'function'
```

Comment:
Will this change after the editor initialized?

For example:
```ts
component: (model) => model.type === 'h1' ? 'affine-h1' : 'affine-paragraph'
```

If I change the model.type from `paragraph` to `h1`, what will happen?

Reply:
The correct behavior should rerender the model with `affine-h1`. But I'm not sure whether or not the editor will do so. Let me write a test.

Reply:
Let's verify this before merging the pr.

Reply:
It works. But you should use `model.type$.value` instead.

Reply:
Cool